tools/emscripten.py

```diff
@@ -507,19 +513,20 @@ def finalize_wasm(infile, outfile, js_syms):
       args.append('--dyncalls-i64')
       # we need to add some dyncalls to the wasm
       modify_wasm = True
+
   if settings.AUTODEBUG:
     # In AUTODEBUG mode we want to delay all legalization until later.  This is hack
     # to force wasm-emscripten-finalize not to do any legalization at all.
     args.append('--bigint')
-  else:
-    if settings.LEGALIZE_JS_FFI:
-      # When we dynamically link our JS loader adds functions from wasm modules to
```

Comment:
This comment was removed becasue it basically duplicate of the one in js_legalization_pass_flags